src/main.cpp

```diff
@@ -969,6 +1050,19 @@ bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransa
         view.SetBackend(dummy);
         }
 
+        // Enforce sequnce numbers as relative lock-time
+        // only for tx.nVersion >= 2.
+        int nLockTimeFlags = 0;
+        if (tx.nVersion >= 2)
+            nLockTimeFlags |= LOCKTIME_VERIFY_SEQUENCE;
+
+        // Only accept nLockTime-using transactions that can be mined in the next
+        // block; we don't want our mempool filled up with transactions that can't
+        // be mined yet.
+        if (LockTime(tx, nLockTimeFlags, &view, chainActive.Height() + 1, GetAdjustedTime()))
```

Comment:
Took me a while to see what you're suggestion, but I agree. I'll rework it to be a policy.h option.